src/Stripe.net/Services/BankAccounts/BankAccountService.cs

```diff
@@ -29,73 +28,72 @@ public BankAccountService(IStripeClient client)
 
         public virtual BankAccount Create(string parentId, BankAccountCreateOptions options, RequestOptions requestOptions = null)
         {
-            return this.CreateNestedEntity(parentId, options, requestOptions);
+            return this.Request<BankAccount>(HttpMethod.Post, $"/v1/customers/{parentId}/sources", options, requestOptions);
```

Comment:
qq: isn't it sometimes true that it uses Account instead of sources?

Reply:
I don't think so. `CreateNestedEntity` [gets the path by calling](https://github.com/stripe/stripe-dotnet/blob/d21cf27c6a19bdcd13a2630b0eb3e8a82900009e/src/Stripe.net/Services/_base/ServiceNested.cs#L31) `this.ClassUrl`, which [ultimately reads the path from](https://github.com/stripe/stripe-dotnet/blob/d21cf27c6a19bdcd13a2630b0eb3e8a82900009e/src/Stripe.net/Services/_base/ServiceNested.cs#L196-L206) `this.BasePath` which is [hardcoded to](https://github.com/stripe/stripe-dotnet/blob/d21cf27c6a19bdcd13a2630b0eb3e8a82900009e/src/Stripe.net/Services/BankAccounts/BankAccountService.cs#L28) `/v1/customers/.../sources`

Reply:
The logic in other libraries e.g. 
https://github.com/stripe/stripe-ruby/blob/master/lib/stripe/resources/bank_account.rb#L28-L31 doesn't apply in `stripe-dotnet` because there are no "instance" methods here that read properties of the returned resource, e.g. you don't do `account_source.delete(...)` or `customer_source.delete(...)` ever, in stripe-dotnet you would

`new ExternalAccountService().delete(...)` if you wanted to `DELETE /v1/accounts/{}/external_accounts` or `new BankAccountService().delete(...)` if you wanted to `DELETE /v1/customers/{}/sources`.